testkit/src/main/java/com/akkaserverless/javasdk/testkit/TestKitEventSourcedEntityContext.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.testkit;
+
+import com.akkaserverless.javasdk.ServiceCallFactory;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntityContext;
+
+public final class TestKitEventSourcedEntityContext implements EventSourcedEntityContext {
```

Comment:
This doesn't have to be public. Can be in `impl` package and written in Scala.

Reply:
I was thinking users _might_ want to create their entity with some specific/custom entity id, but I guess we can provide factories for that as well, from the testkit class itself for example.

Reply:
you have already defined a second `of` with entityId 😄

Reply:
🤦 problem solved then, I'll make it private!